doc/ale.txt

```diff
@@ -46,6 +46,8 @@ CONTENTS                                                         *ale-contents*
       gcc.................................|ale-fortran-gcc|
     fusionscript..........................|ale-fuse-options|
       fusion-lint.........................|ale-fuse-fusionlint|
+    glsl..................................|ale-glsl-options|
+      glslang.............................|ale-glsl-glslang|
```

Comment:
The Bash script pointed out that there's no tag for this doc tag link `|ale-glsl-glslang|`. I think the name is right here. You should make the heading name and the tag in `doc/ale-glsl.txt` match what you have here.

Reply:
Oh yeah I was apparently moving these things around at some point. The name here is the one I intended.